src/pages/confirmAppointment/[appointment_id].tsx

```diff
@@ -53,74 +110,165 @@ export default function Home() {
+                  errMsg=""
+                  readOnly={true}
+                />
+              </Box>
+
+              {/* Location */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <LocationTextField
+                  header="Location"
+                  placeholder="Enter Location"
+                  initialValue={appointment.detailHeader.location}
+                  onChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  readOnly={true}
+                />
+              </Box>
+
+              {/* Date & Time */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <DisplayDateTime
+                  header="Date & Time"
+                  value={`${appointment.detailHeader.startDateTime} - ${appointment.detailHeader.endDateTime}`}
+                  readOnly={true}
+                />
+              </Box>
+
+              {/* Tags */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <Tags
+                  header="Tag"
+                  value={appointment.detailHeader.tags.map((t, index): Tag => {
+                    return { name: t, id: index };
+                  })}
+                  handleValueChange={() => {}}
+                  menuValue={[]}
+                  isErr={false}
+                  errMsg=""
+                  readOnly={true}
+                />
+                <FormHelperText>{"\u00A0"}</FormHelperText>
+              </Box>
+
+              {/* Description */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <DescriptionTextField
+                  name="description"
+                  header="Description"
+                  placeholder="Enter Description Here"
+                  value={appointment.detailHeader.description}
+                  handleValueChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  height={8}
+                  readOnly={true}
+                />
+              </Box>
+            </Stack>
+          </Card>
+          <Card sx={ConfirmAppointmentStyle.Card}>
+            <Stack spacing={3} alignItems="center" justifyContent="center">
+              {/* Image list */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <DisplayImages header="Image" images={appointment.images} />
+              </Box>
+
+              {/* Number of participants */}
+              <Box sx={ConfirmAppointmentStyle.TextField}>
+                <Typography variant="h3">{`Number of Participant : ${appointment.acceptParticipants.length}`}</Typography>
+              </Box>
+
+              {/* Participant List */}
+              <Stack
+                spacing={0.5}
+                alignItems="start"
+                justifyContent="center"
+                sx={ConfirmAppointmentStyle.TextField}
+              >
+                <Typography variant="h2">Join with</Typography>
```

Comment:
I think this one should be `Participant List`

Reply:
Opps, solved krub

Reply:
I don't see the change na. Did you forget to commit?

Reply:
forgot 😢

Reply:
Committed